todo-app/src/TodoList.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+import {observer, inject} from 'mobx-react';
+import {todoStore} from './stores';
+
+@inject("todoStore")
+@observer
+export default class TodoList extends React.Component {
+
+  state = {
+    text: ''
+  }; 
+
+
+  addTodo = (text) => {
+    text = this.state.text
+    todoStore.addTodo(text)
+  };
+
+  onAddChange = event => {
+    var text = event.target.value;
+    this.setState({
+      text: text
+    });
+  };
+
+  deleteTodo = (todoID) => {
+    todoStore.onDelete(todoID); 
+  } 
+  
+  changeStatus = (todoID) => {
+   todoStore.onDone(todoID);
+  };
+
+
+  render() {
+    if(todoStore.todos.length === 0){
```

Comment:
When I load the page, it says "No items..." while the todos are loaded from the API. This is not a great user experience.